homeassistant/components/vodafone_station/config_flow.py

```diff
@@ -139,6 +139,47 @@ async def async_step_reauth_confirm(
             errors=errors,
         )
 
+    async def async_step_reconfigure(
+        self, user_input: dict[str, Any] | None = None
+    ) -> ConfigFlowResult:
+        """Handle reconfiguration of the device."""
+        reconfigure_entry = self._get_reconfigure_entry()
+        if not user_input:
+            return self.async_show_form(
+                step_id="reconfigure", data_schema=user_form_schema(user_input)
+            )
+
+        updated_host = user_input[CONF_HOST]
+
+        if reconfigure_entry.data.get(CONF_HOST) != updated_host:
```

Comment:
When would `CONF_HOST` not be present?

```suggestion
        if reconfigure_entry.data[CONF_HOST] != updated_host:
```